src/daemon-command/script.ts

```diff
@@ -37,18 +49,54 @@ export class Script {
         }
     }
 
-    handleLogs(data: Buffer): void {
-        this.logSockets.forEach((socket) => {
-            socket.write(`${this.name}: ${data}`);
-        });
+    handleLogs(data: Buffer | string): void {
         const incomingLines = data.toString().split("\n");
         if (incomingLines[incomingLines.length - 1] == "") incomingLines.splice(incomingLines.length - 1, 1);
+        for (const line of incomingLines) {
+            console.log(`${this.name}: ${line}`);
+            this.logSockets.forEach((socket) => {
+                socket.write(`${this.name}: ${line}`);
+            });
+        }
         const removeLines = incomingLines.length - (KEEP_LOG_LINES - this.logBuffer.length);
         this.logBuffer.splice(0, removeLines > 0 ? removeLines : 0, ...incomingLines);
     }
 
-    startProcess(): void {
-        console.log(`${colors.bgGreen.bold.black(" dev-pm ")} starting: ${this.scriptDefinition.script}`);
+    async startProcess(): Promise<void> {
+        if (this.waitOn.length > 0) {
+            this.status = "waiting";
+
+            try {
+                //first a silent try for 200ms (to avoid too much console output)
+                await waitOn({
+                    resources: this.waitOn,
+                    timeout: 1000,
```

Comment:
shouldn't this be 200?

Reply:
Or did you mean to add a `delay: 200`?

Reply:
yes, the comment is outdated, removed the value

(no, delay does something else)

Reply:
ok but then the logs below are also wrong

Reply:
oops